server/graphql/v2/object/ContributionStats.js

```diff
@@ -0,0 +1,12 @@
+import { GraphQLFloat, GraphQLInt, GraphQLObjectType } from 'graphql';
+
+export const ContributionStats = new GraphQLObjectType({
+  name: 'ContributionStats',
+  description: 'Contribution statistics related to the given accounts',
+  fields: () => ({
+    numContributions: { type: GraphQLInt, description: 'The total number of contributions' },
+    numOneTime: { type: GraphQLInt, description: 'Number of one time contributions' },
+    numRecurring: { type: GraphQLInt, description: 'Number of recurring contributions' },
+    dailyAvgIncome: { type: GraphQLFloat, description: 'The daily average income' },
```

Comment:
```suggestion
    dailyAvgIncome: { type: new GraphQLNonNull(Amount), description: 'The daily average income' },
```

Reply:
```suggestion
    dailyAverageIncome: { type: new GraphQLNonNull(Amount), description: 'The daily average income' },
```

Never abbreviations like 'avg' please, full words.